.gitignore

```diff
@@ -0,0 +1,107 @@
+# Logs
```

Comment:
Please remove, each example defines its own `.gitignore` so this shouldn't be needed at the root.